tests.py

```diff
@@ -1,17 +1,25 @@
-from app.views import app
+from .app.views import app
 import json
 
 from base import BaseClass
 
 class UserTestCase(BaseClass):
     """This class represents the user test cases."""
 
-    def test_signup(self):
-        """Test API can successfully register a new user (POST request)"""
+    # def test_signup(self):
```

Comment:
it's not advisable to comment out unused code. Just remove it

Reply:
okay